Header header

float32 thrust
float32 roll
float32 pitch
float32 yaw_delta

float32 kp_roll
float32 kd_roll

float32 kp_pitch
float32 kd_pitch

float32 kd_yaw

float32 p_des
float32 q_des
float32 r_des

float32 roll_delta
float32 pitch_delta

float32 vicon_roll
float32 vicon_pitch
float32 vicon_bil